user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -56,6 +56,8 @@ Method Signature Changes
 Enhancements
 ************
 
+- The Debug Toolbar now has a new "Hot Reload" feature that can be used to automatically reload the page when a file is changed. See :ref:`debug-toolbar-hot-reload`.
+
```

Comment:
**Enhancements** are categorized due to the large number of items.

This item is categorized as **Testing** according to the User's Guide:
https://github.com/codeigniter4/CodeIgniter4/blob/4.4/user_guide_src/source/changelogs/v4.4.0.rst#testing
Because "The Debug Toolbar" is described under **Testing**:
https://codeigniter4.github.io/CodeIgniter4/testing/debugging.html